src/main/java/marquez/api/resources/JobResource.java

```diff
@@ -149,12 +151,25 @@ public Response create(
         .build();
   }
 
-  @Timed
-  @ResponseMetered
-  @ExceptionMetered
   @GET
-  @Path("/jobs/runs/{id}")
   @Produces(APPLICATION_JSON)
+  @Path("/namespaces/{namespace}/jobs/{job}/runs")
+  public Response getRunsForJob(
+      @PathParam("namespace") final NamespaceName namespaceName,
+      @PathParam("job") final String job,
+      @QueryParam("limit") @DefaultValue("100") Integer limit,
+      @QueryParam("offset") @DefaultValue("0") Integer offset)
+      throws MarquezServiceException {
+    if (!namespaceService.exists(namespaceName)) {
+      return Response.status(Response.Status.NOT_FOUND).entity("Namespace not found").build();
+    }
+    final List<JobRun> jobRuns = jobService.getAllRunsOfJob(namespaceName, job, limit, offset);
+    return Response.ok().entity(JobRunResponseMapper.map(jobRuns)).build();
+  }
+
+  @GET
+  @Timed
+  @Path("/jobs/runs/{id}")
```

Comment:
_Minor_: We'll want to keep the metric annotations for `GET` `/jobs/runs/{id}`. May have been removed when merging with `master`:

```java
@Timed
@ResponseMetered
@ExceptionMetered
@GET
@Path("/jobs/runs/{id}")
@Produces(APPLICATION_JSON)
public Response get(@PathParam("id") final UUID runId) throws MarquezServiceException {
  . . .
}
``

Reply:
yah was a bad merge, should've checked the diff. fixing it

Reply:
fixed